services/examples/pool-client/README.md

```diff
@@ -0,0 +1,40 @@
+# Overview
+
+The pool-client is an example of how to interact with the Pool service over a
+message bus.
+
+It performs the following operations:
+
+1. Creates a pool
+2. Lists the pools
+3. Destroys the previously created pool
+4. Lists the pools again
+
+## Running the example
+
+The pool-client example requires the following to be started inside the nix shell:
+
+1. nats server
+```bash
+nats-server
+```
+
+2. mayastor - specifying the message bus endpoint and node name as "mayastor-node"
+```bash
+sudo ./target/debug/mayastor -n 127.0.0.1:4222 -N mayastor-node
+```
+
+2. node service
```

Comment:
might want to start the node service before mayastor, because mayastor sends registration messages every N seconds, which means if you start up the node and pool service too quickly, you might have not received the registration yet.

Reply:
Thanks, I've reordered the steps to put the node service before mayastor.